### DISCUSSION TITLE
Removing Stitching artefact in EBSD data

### DISCUSSION THREAD

**Discussion by fsamaraie** [mtex-toolbox/mtex#1718] [Ask Anything]
Hello Mtex Community!
Lately I started using mtex toolbox and I have been enjoying it. Im reconstructing some EBSD data and I am faced by a problem. The data that I have are a subset of a large-area map, in which tile stitching was performed manually. Consequently, there are some artefacts such as vertical lines that reflect low-angle boundaries at locations where  tiles are stitched (please see the image below). I was wondering if you can direct me to a function or a tool that can help in solving this issue. I looked at the twinning merging in the documentation, but it seems to be concerned with twinning rather than artefact. 
Any help would be appreciated. 

![image](https://user-images.githubusercontent.com/115000993/212473626-c5366cb5-5724-4fd5-8f15-4e0e226ebdff.png)

**Comment by kilir** [comment]
Hi,
this is a known issue due to an incorrect orientation solution in a large tile which manifests in an apparent orientation gradient, unrelated to stitching. Reasons can be manifold, e.g. not properly mounted sample or  indexing software not honoring the projection center properly or a combination of some.

Under the assumption that neither redoing the map with a better sample mounting, smaller tile size or asking the vendor to fix the indexing software is an option, you can try to sort out the wrong boundaries. Note that intragranular orientation variations and related measures (e.g. GOS, KAM,...) will still be wrong. I'd probably look for low angle boundaries with a) a specific angle b) direction, maybe curvature to figure out which ones are straight, horizontal or vertical and run over a long distance. Additionally limiting this selection to certain domains of the map might be helpful.

Cheers,
Rüdiger
